### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by ibio-project** [sqlitebrowser/sqlitebrowser#3825]
### What did you do?

I tried to use string_agg function like:

select id, string_agg(item, '; ') as items
from table1 group by 1;

### What did you expect to see?

Aggregated result like:

1 | aa; bb; cc
2| dd; ee
  ;

### What did you see instead?

Reseult: no such function: string_agg.

The SQL works perfectly on command line but it failed on DB4S.
If string_agg is replaced with group_concat, it worked perfectly.
I suspect syntax checking somehow fails to identify string_agg as valid function name.

### DB4S Version

3.12.2

### What OS are you seeing the problem on?

Windows

### OS version

Windows 11

### Relevant log output

```shell
Reseult: no such function: string_agg.
```

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by chrisjlocke** [comment]
String_agg was introduced in Oct 2023 so it's probably not in the sqlite version in 3.12.2
Have you used v3.13.1 of the application? This would use a newer version of sqlite.

**Comment by mgrojo** [comment]
Even in 3.13, it was missing in the call tips of the SQL editor. Now, I've added a bunch of them that are on the latest SQLite version but were also missing: `concat` `concat_ws` `format` `likelihood` `likely`  `octet_length` `unhex` `unlikely` `string_agg`

**Comment by Talkless** [comment]
So this issue can be closed due to https://github.com/sqlitebrowser/sqlitebrowser/commit/3866328111a33d0ca2a72bb61d3335ac649c6c4d ?

**Comment by mgrojo** [comment]
Yes, I guess this can be closed. If anyone has still a glitch to declare, feel free to do it, even if the issue was closed.
